tests/integration/testlib/bondlib.py

```diff
@@ -43,6 +43,10 @@ def bond_interface(name, slaves, extra_iface_state=None, create=True):
     }
     if extra_iface_state:
         desired_state[Interface.KEY][0].update(extra_iface_state)
+        if slaves:
+            desired_state[Interface.KEY][0][Bond.CONFIG_SUBTREE][
+                Bond.SLAVES
+            ] = slaves
```

Comment:
But we already set them on line 39.

Reply:
But if `extra_iface_state` is set, it will override slaves when updating.

Reply:
ack